ignite/services/network/network.go

```diff
@@ -43,7 +43,7 @@ type Network struct {
 	stakingQuery            stakingtypes.QueryClient
 	bankQuery               banktypes.QueryClient
 	monitoringConsumerQuery monitoringctypes.QueryClient
-	monitoringProviderQuery monitoringptypes.QueryClient
+	monitoringProviderQuery monitoringptypes.QueryClient //nolint:unused
```

Comment:
looks like the unused nolint is unused lol

Reply:
I don't think it will show up that way after bumping go and golangci-lint.

Why bump them?

Basically, opens up a whole new world of rigor...